source/Calamari.Common/Features/Behaviours/ConfigurationVariablesBehaviour.cs

```diff
@@ -25,7 +25,10 @@ public ConfigurationVariablesBehaviour(ICalamariFileSystem fileSystem, IConfigur
 
         public bool IsEnabled(RunningDeployment context)
         {
-            return context.Variables.GetFlag(KnownVariables.Package.AutomaticallyUpdateAppSettingsAndConnectionStrings);
+            var features = context.Variables.GetStrings(KnownVariables.Package.EnabledFeatures)
```

Comment:
Looks like we would be calling `GetStrings()` and `features.Contains()` in a few places, how about making this an extension method so we can call `context.Variables.IsFeatureEnabled(KnownVariables.Features.ConfigurationVariables)`?